composer.json

```diff
@@ -33,7 +33,6 @@
         "league/flysystem-aws-s3-v3": "~1.0",
         "league/flysystem-azure": "~1.0",
         "league/flysystem-cached-adapter": "~1.0",
-        "league/flysystem-copy": "~1.0",
```

Comment:
Why did you remove this?

Reply:
https://github.com/thephpleague/flysystem-copy/issues/16

TLDR - This service does not exist since 2016